code/mission/missionparse.cpp

```diff
@@ -1841,7 +1841,7 @@ int parse_create_object(p_object *pobjp)
 		// if the leader will be destroyed before the mission starts, then *only* create the leader;
 		// don't create the rest of the group (this is what retail did)
 		if (pobjp->destroy_before_mission_time >= 0)
-			return parse_create_object_sub(pobjp);
+			return parse_create_object_sub(pobjp, standalone_ship);
```

Comment:
This one can be removed.  If we don't need to specify the flag in the dock helper because it'll never be docked, then we don't need to specify it here because it'll never be destroyed before mission.